### ISSUE TITLE
Can't set cgame VM type in non-cheat game

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#1190]
I get this while compiling `0.52.0/sync` and run the game with nexe:

```
Warn: Error during error handling: CGame VM: Reader: Unread bytes at end of message
```

I don't know if it's an issue on my end or our game is borked.

**Comment by illwieckz** [comment]
Hmmm, I see a warning that is meant to be fixed. How can I make sure I don't load nexe from a dpk but from filesytem?

**Comment by illwieckz** [comment]
Yes it is using the 0.51.1 nexe, my incoming 0.52.0 packages just extend the 0.51.1 packages.

**Comment by illwieckz** [comment]
The issue happens when I try to load a map with `map`, because I want to test with cheats disabled.

**Comment by slipher** [comment]
So the question was how to load with nonzero `vm.cgame.type` but with cheats off, for testing purposes. You can't do that but I'd say this was solved by moving buildable cheat behavior to `g_instantBuilding` so that devmap can be used for testing any gameplay mechanics without interference. (Also `g_persistDevmap` for completeness, if you need to test map changes.)
